content/flux/v0.x/query-data/sql/sqlite.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Query SQLite
+list_title: SQLite
+description: >
+  Use [`sql.from()`](/flux/v0.x/stdlib/sql/from/) with the `sqlite3` driver to query SQLite.
+menu:
+  flux_0_x:
+    name: SQLite
+    parent: SQL databases
+weight: 101
+related:
+  - /flux/v0.x/stdlib/sql/from/
+list_code_example: |
+  ```js
+  import "sql"
+
+  sql.from(
+    driverName: "sqlite3",
+    dataSourceName: "file:/path/to/example.db?cache=shared&mode=ro",
+    query: "SELECT * FROM example_table"
+  )
+  ```
+---
+
+To query [SQLite](https://www.sqlite.org/index.html) with Flux, import the
+[`sql` package](/flux/v0.x/stdlib/sql/) and use the [`sql.from()` function](/flux/v0.x/stdlib/sql/from/)
+with the `sqlite3` driver.
+Provide the following parameters:
+
+- **driverName**: sqlite3
+- **dataSourceName**: [SQLite data source name (DSN)](#data-source-name)
+  _(also known as **connection string**)_
+- **query**: SQL query to execute
+
+```js
+import "sql"
+
+sql.from(
+  driverName: "sqlite3",
+  dataSourceName: "file:/path/to/example.db?cache=shared&mode=ro",
+  query: "SELECT * FROM example_table"
+)
+```
+
+{{% note %}}
+#### Requires file system access
+To query SQLite, Flux must have access to the filesystem.
+If Flux does not have access to the file system, the query will return an error
+similar to one of the following:
+
+- `Error: unable to open database file`
+- `failed to read file: filesystem service is uninitialized`
+- `An internal error has occurred`
+
+If using InfluxDB Cloud or InfluxDB OSS, the Flux process **does not** have 
+access to the filesystem.
```

Comment:
Maybe a note about how you must use the REPL to do this?

Reply:
The REPL isn't the only other way to run Flux. Users can build their own implementations, so I hesitate to call the REPL out specifically here.